bids2table/_indexing.py

```diff
@@ -400,7 +402,14 @@ def _index_bids_subject_dir(
     _, subject = path.name.split("-", maxsplit=1)
 
     records = []
-    for p in path.rglob("sub-*"):
+    if isinstance(path, CloudPath):
```

Comment:
or python >3.13 right?

Reply:
This is fixed in [1f77700](https://github.com/childmindresearch/bids2table/pull/56/commits/1f777004efd4e7b2403bdf6c2f55710536f08f3d). 

I did need to have separate `rglob` calls for `ClouthPath` and `py3.13+` - the arguments were a little bit different.